### PR TITLE
Made changes to export metrics for gcs read calls along with type

### PR DESCRIPTION
**Pull request by vadlakondaswetha**
1. Added two new metrics - gcs/read_count to capture number of read calls along with type, gcs/downloaded_bytes_count to capture amount of data downloading from gcs.
2. gcsfuse_read_bytes is same as gcs/read_bytes_count. Hence removed gcsfuse_read_bytes

### CODE REVIEW

File: internal/gcsx/random_reader.go [hunk 1]
```diff
@@ -47,15 +54,27 @@ const maxReadSize = 8 * MB
...
 // Initialize the metrics.
 func init() {
-	v := &view.View{
-		Name:        "gcsfuse_read_bytes",
-		Measure:     readBytes,
-		Description: "The number of bytes read from GCS",
-		Aggregation: view.Sum(),
-	}
-	if err := view.Register(v); err != nil {
+	if err := view.Register(
```
Comment (avidullu) [inline internal/gcsx/random_reader.go:63]: nit: formatting looks off.

File: internal/gcsx/random_reader.go [hunk 2]
```diff
@@ -350,5 +370,25 @@ func (rr *randomReader) startRead(
 	rr.start = start
 	rr.limit = end
 
+	captureMetrics(ctx, rr.seeks)
```
Comment (avidullu) [inline internal/gcsx/random_reader.go:373]: why pass the seeks here? Can we make a readType variable in this function, populate it based on the conditions in this function and then pass it to the captureMetrics?

Comment (avidullu) [inline internal/gcsx/random_reader.go:373]: I think we can also record the size (end - start) of the read? Or is that been done elsewhere?

File: internal/gcsx/random_reader.go (hunk 1 above)
Comment (vadlakondaswetha) [reply internal/gcsx/random_reader.go:63]: It looks proper in the intellij: https://screenshot.googleplex.com/FAhYcbCsieQ9t5z.png

File: internal/gcsx/random_reader.go (hunk 2 above)
Comment (vadlakondaswetha) [reply internal/gcsx/random_reader.go]: We have gcsfuse_read_bytes metrics which captures the read bytes. Not based on data fetched from gcs but data read by user.  do you want to capture the amount of data fetched from gcs?

Comment (avidullu) [reply internal/gcsx/random_reader.go]: right, that's the intention.

File: internal/gcsx/random_reader.go (hunk 1 above)
Comment (avidullu) [reply internal/gcsx/random_reader.go:63]: I see it as the attached image, hence the comment.

codeformat
t

**Review by avidullu — APPROVED** [review]

**Pull request merged**